engine/src/main/java/org/terasology/rendering/opengl/PBO.java

```diff
@@ -30,7 +30,8 @@
  * Created by manu on 15.03.2015.
  */
 public class PBO {
-    private int pboId;
+    private ByteBuffer resultBuffer = null;
```

Comment:
re-uses the buffer if it's the right size, avoiding:
``   ByteBuffer resultBuffer = BufferUtils.createByteBuffer(cachedBuffer.capacity());